tedge_config/src/tedge_config_defaults.rs

```diff
@@ -47,6 +50,7 @@ impl From<&TEdgeConfigLocation> for TEdgeConfigDefaults {
                 .into(),
             default_azure_root_cert_path: system_cert_path.clone().into(),
             default_c8y_root_cert_path: system_cert_path.into(),
+            default_mosquitto_port: 1883,
```

Comment:
This default seems to be unused, since one has a `DEFAULT_PORT` in `mqtt/cli.rs`.

Reply:
I'd rather prefer to keep this and remove DEFAULT_PORT and match-sentence in `mqtt/cli.rs`.